fineftp-server/src/filesystem.cpp

```diff
@@ -209,20 +209,20 @@ namespace Filesystem
       "Dec"
     };
 
-    // Use strftime for the day and year / time
-    char date[80];
+    std::stringstream date;
+
     if (file_year == current_year)
     {
       // We are allowed to return the time!
-      strftime(date, sizeof(date), " %e %R", &file_timeinfo);
+      date << " " << file_timeinfo.tm_mday << " " << file_timeinfo.tm_hour << ":" << file_timeinfo.tm_min;
```

Comment:
Please change, so this outputs the time with leading space (for hours) and leading 0 (for minutes) in case of a single-digit values, e.g. with [std::setw](https://en.cppreference.com/w/cpp/io/manip/setw) and [std::setfill](https://en.cppreference.com/w/cpp/io/manip/setfill)

```
<time> ::= <digit> <digit> ":" <digit> <digit>
         |   SPC   <digit> ":" <digit> <digit>
```

https://files.stairways.com/other/ftp-list-specs-info.txt

While omitting spaces and zeros may work with a lot of FTP clients, staying as close to that format increases compatibility.